### DISCUSSION TITLE
Change the color of the map XD

### DISCUSSION THREAD

**Discussion by iBLiSSIN** [IceHacks/SurvivCheatInjector#749] [Ideas]
I know it posible, so pls do it :))

**Comment by AsiaGuy** [comment]
what color?

**Comment by iBLiSSIN** [comment]
> what color?

Custom color >:) (maybe are dark/light green, white, …. or custom with input color code)

**Comment by AsiaGuy** [comment]
hmm i don't think it's a good idea because the hack user will have eyes pain

**Comment by iBLiSSIN** [comment]
> hmm i don't think it's a good idea because the hack user will have eyes pain

Hmm,you can disable it :>

**Comment by AsiaGuy** [comment]
is it default or not

**Comment by ghost** [comment]
> is it default or not

@AsiaGuy, it's most likely not going to be the default option. I wouldn't want it to be that way as well. It _should_ be something that can be enabled by **choice**.

**Comment by iBLiSSIN** [comment]
> > is it default or not
> 
> @AsiaGuy, it's most likely not going to be the default option. I wouldn't want it to be that way as well. It _should_ be something that can be enabled by **choice**.

Yea 
Or I can change it with edit the hack file XD

**Comment by AsiaGuy** [comment]
bruhh

**Comment by iBLiSSIN** [comment]
Lmao
